group_vars/gxconfig.yml

```diff
@@ -2243,6 +2284,18 @@ base_app_main: &BASE_APP_MAIN
   # handler processes. Float values are allowed.
   #workflow_monitor_sleep: 1.0
 
+  # In which cases Galaxy should calculate a hash for a new dataset.
+  # Dataset hashes can be used by the Galaxy job cache/search to check
+  # if job inputs match. Setting the 'enable_celery_tasks' option to
+  # true is also required for dataset hash calculation. Possible values
+  # are: 'always', 'upload' (the default), 'never'. If set to 'upload',
+  # the hash is calculated only for the outputs of upload jobs.
+  #calculate_dataset_hash: upload
```

Comment:
I guess we should start with upload? Mid-term I would like to enable all ... but it only makes sense if we use those as well.

Reply:
I don't understand the connection with TIaaS, but lets discuss this in the next weeks. At first I would like to test a few GTN tuts and see how good the cache is.